pkg/kubelet/kubelet_getters.go

```diff
@@ -196,18 +196,6 @@ func (kl *Kubelet) getRuntime() kubecontainer.Runtime {
 
 // GetNode returns the node info for the configured node name of this Kubelet.
 func (kl *Kubelet) GetNode() (*v1.Node, error) {
-	if kl.kubeClient == nil {
```

Comment:
`GetNode()` and `getNodeAnyWay()` are not equivalent. `getNodeAnyWay()` will return `initialNode()` if for some reason, `GetNodeInfo` returns an error. 

Could you check all the call sites to make sure they are actually interchangeable?